src/components/Button/ButtonUser/ButtonUser.scss

```
@@ -0,0 +1,26 @@
+.btn-user {
+    height: 60px;
+    background-color: rgba(34, 49, 170, 1);
+    // border: none;
+    font-size: 14px;
+    // font-weight: 600;
+    display: flex;
+
+    & p {
```

Знак & лишний. И лучше использовать классы для стилизации элементов
```suggestion
    p {
```